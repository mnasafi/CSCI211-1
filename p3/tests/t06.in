insert
United Break Guitars
http://www.youtube.com/watch?v=5YGc4zOqozo
Great example of one person getting a giant company to listen
4.5
5
insert
Funny Cats
http://www.youtube.com/watch?v=nTasT5h0LEg
Why are there so many cat videos on youtube?
2.75
1
insert
Dog Playing Piano
http://www.youtube.com/watch?v=yT64KGSuLK4
My daughter posted this, so I had to include it
1.3
2
insert
Cupcake eating challenge
http://www.youtube.com/watch?v=kzZXOSQ5GjE
You should avoid cupcake eating contests
1.5
3
lookup
United Break Guitars
lookup
Funny Cats
lookup
Dog Playing Piano
lookup
Cupcake eating challenge
