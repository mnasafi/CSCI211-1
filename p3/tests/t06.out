United Break Guitars, http://www.youtube.com/watch?v=5YGc4zOqozo, Great example of one person getting a giant company to listen, 4.5, *****
Funny Cats, http://www.youtube.com/watch?v=nTasT5h0LEg, Why are there so many cat videos on youtube?, 2.75, *
Dog Playing Piano, http://www.youtube.com/watch?v=yT64KGSuLK4, My daughter posted this, so I had to include it, 1.3, **
Cupcake eating challenge, http://www.youtube.com/watch?v=kzZXOSQ5GjE, You should avoid cupcake eating contests, 1.5, ***
